### PR TITLE
First refactoring

### PR DESCRIPTION
**Pull request by bortoletti-giorgia**
- Divided classes in packages (location, alarm, activity, contacts, deprecated)
- Added two parser to compose request and response messages for alarm and location

### CODE REVIEW

File: app/src/main/java/ingsw/group1/findmyphone/Constants.java [hunk 1]
```diff
@@ -4,8 +4,7 @@
  * Contains the constants that need to be shared between activities of the application,
  * all of them are defined package-private
  */
-class Constants {
-    final int request = 0, response = 1;
-    final String receivedStringMessage = "receivedStringMessage";
-    final String receivedStringAddress = "receivedStringAddress";
+public final  class Constants {
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/Constants.java]: For constants capital snake case is usually regarded as better, i.e.:
`RECEIVED_STRING_MESSAGE`

File: app/src/main/java/ingsw/group1/findmyphone/Manager.java [hunk 2]
```diff
@@ -4,149 +4,146 @@
...
 import ingsw.group1.msglibrary.SMSManager;
 import ingsw.group1.msglibrary.SMSMessage;
 import ingsw.group1.msglibrary.SMSPeer;
 
-/***
+/**
+ * This is the main manager
+ *
  * @author Pardeep
- * Class
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/Manager.java:14]: Mmmh yes this is a Class 🧐

File: app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java [hunk 3]
```diff
@@ -0,0 +1,36 @@
+package ingsw.group1.findmyphone.alarm;
+
+import android.location.Location;
+
+/**
+ * Manager that provides composition of message for alarm request and response
+ * Message request is composed as:
+ *  ALARM_REQUEST
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java]: I believe something might be missing here

File: app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+ *
+ * @author Giorgia Bortoletti
+ */
+public class ParserAlarmMessage {
+
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java]: I believe `AlarmMessageParser` would sound nicer

File: app/src/main/java/ingsw/group1/findmyphone/location/ParserLocationMessage.java [hunk 5]
```diff
@@ -0,0 +1,58 @@
...
+/**
+ * Manager that provides composition of message for alarm and location request and response
+ * Message request is composed as:
+ *  LOCATION_REQUEST
+ * Message response is composed in this way:
+ *  LOCATION_RESPONSE<LT>value_latitude</LT><LG>value_longitude</LG>
+ *
+ * @author Giorgia Bortoletti
+ */
+public class ParserLocationMessage {
```
Comment (ricdezen) [inline app/src/main/java/ingsw/group1/findmyphone/location/ParserLocationMessage.java]: Same as Alarm Parser

**Review by ricdezen — COMMENTED** [review]
I would need some time to properly review the architecture of the code, but I'm running kinda short right now, I only pointed out something about the style

File: app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java (hunk 3 above)
Comment (bortoletti-giorgia) [reply app/src/main/java/ingsw/group1/findmyphone/alarm/ParserAlarmMessage.java]: For the actual composition of alarm request, the message is only 'ALARM_REQUEST'. We have to talk and decide about it.

**Comment by bortoletti-giorgia** [comment]
No problem. I have to write all the tests.

**Pull request closed without merging**